examples/src/main/scala/example/ExampleService.scala

```diff
@@ -17,19 +17,19 @@ object ExampleService {
   }
 
   def getCharacters(origin: Option[Origin]): URIO[ExampleService, List[Character]] =
-    URIO.serviceWithZIO(_.getCharacters(origin))
+    ZIO.serviceWithZIO(_.getCharacters(origin))
 
   def findCharacter(name: String): URIO[ExampleService, Option[Character]] =
-    URIO.serviceWithZIO(_.findCharacter(name))
+    ZIO.serviceWithZIO(_.findCharacter(name))
 
   def deleteCharacter(name: String): URIO[ExampleService, Boolean] =
-    URIO.serviceWithZIO(_.deleteCharacter(name))
+    ZIO.serviceWithZIO(_.deleteCharacter(name))
 
   def deletedEvents: ZStream[ExampleService, Nothing, String] =
     ZStream.serviceWithStream(_.deletedEvents)
 
-  def make(initial: List[Character]): ZLayer[Any, Nothing, ExampleService] =
-    (for {
+  def make(initial: List[Character]): ZLayer[Any, Nothing, ExampleService] = ZLayer.fromZIO {
```

Comment:
Can this be just `ZLayer {`? Or is there a difference?

Reply:
Looks like it could be.